workspace/welcome-to-xod/111-internet/patch.xodp

```diff
@@ -0,0 +1,171 @@
+{
+  "comments": [
+    {
+      "content": "Now you got a remote analog gauge for the data that you send to the cloud. You can place it on your desktop and watch for the moisture on your greenhouse in the countryside.\n\nAlso, another device might turn on or off the water pump, depending on the same data. Moreover, you can send a status of a water pump to the cloud and add a few nodes here to indicate that the water pump is working now.",
```

Comment:
Now you got a remote analog gauge for the data that you send to the cloud. You can place it on your desktop and watch for the moisture on your greenhouse in the countryside.

Also, another device might turn on or off the water pump, depending on the same data. Moreover, you can send a status of a water pump to the cloud and add some nodes here to indicate that the water pump is working.